db/migrate/20160927022134_create_notifications.rb

```diff
@@ -0,0 +1,18 @@
+class CreateNotifications < ActiveRecord::Migration
+  def change
+    create_table :notifications do |t|
+      t.belongs_to :event, index: true, foreign_key: true
+      t.string :status
+      t.string :subject
+      t.string :body
+      t.integer :hours_to_try
+      t.integer :minutes_interval
+      t.integer :attempts_before_escalation
+      t.datetime :start_at
+      t.datetime :started_at
+      t.string :channels
```

Comment:
Since we'll probably want to store the channels as a serialized array, I think this should be text
http://stackoverflow.com/questions/6694432/using-rails-serialize-to-save-hash-to-database